ui/css/perf.css

```diff
@@ -267,6 +293,19 @@ th {
   font-size: inherit;
 }
 
+.compare-table .text-success {
+  color: #1e8836 !important;
```

Comment:
I don't think we'd need to change these green and red colors since we're already using icons with them.

Reply:
I may be looking at the wrong part, but this changes the "untriaged"/"invalid" text in the Compare table, which does not have an icon yet.
I have inserted to show where it is changed:
![image](https://user-images.githubusercontent.com/3901809/72012189-45471600-323a-11ea-9c29-23d324f19b8e.png)

I can insert an icon and keep the color as it was before.

Reply:
It also changes the thumbs up icon that appears for certain results in the compare view table [ here](https://treeherder.mozilla.org/perf.html#/compare?originalProject=mozilla-central&originalRevision=adeaa48edc50404d3c77a2faaf496ac89711b8f6&newProject=mozilla-central&newRevision=a4592adb2fe04393c1a5df9487ff88a70eeb0881&framework=1). Click on the "Show only important button".

Visually the shade looks slightly different than the default `bg-success` color used by order of magnitude bar (same color as `text-success`). 

I prefer to create a custom class name to assign to elements where needed instead of making a change like this to all ancestors of an element and overriding a bootstrap style.

Reply:
And yes, the icon is oK. I think where it makes sense we should use icons for use with red/green instead of trying to get the shades perfect. For someone with red/green color blindess, I don't think the shade change will make a difference.